pkg/scheduler/internal/queue/scheduling_queue_test.go

```diff
@@ -768,98 +768,6 @@ func TestRecentlyTriedPodsGoBack(t *testing.T) {
 	}
 }
 
-// TestPodFailedSchedulingMultipleTimesDoesNotBlockNewerPod tests
-// that a pod determined as unschedulable multiple times doesn't block any newer pod.
-// This behavior ensures that an unschedulable pod does not block head of the queue when there
-// are frequent events that move pods to the active queue.
-func TestPodFailedSchedulingMultipleTimesDoesNotBlockNewerPod(t *testing.T) {
```

Comment:
Previously, I deleted this test because it was introduced through the previous pr. But now I think this test can be retained. I made some modifications. Thank you.